src/Symfony/Component/Form/Extension/Core/Type/FormType.php

```diff
@@ -194,6 +194,7 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
             'read_only'          => false,
             'max_length'         => null,
             'pattern'            => null,
+            'map_to'             => null,
```

Comment:
The "map_to" option should receive the value of the "property_path" option by default. See how the options "inherit_data"/"virtual" are handled in FormType.

You should also add a normalizer for the "property_path" option that triggers a deprecation error if the option is set. See how "choice_list" is handled in ChoiceType for that.